easy-ui-react/src/InputField/InputField.module.scss

```diff
@@ -162,3 +162,44 @@
 .caption {
   margin-top: design-token("space.0.5");
 }
+
+.textStartInput {
+  @include Input.textStartInput;
+}
+
+.textEndInput {
+  @include Input.textEndInput;
+}
+
+.text {
+  border: 0;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  position: absolute;
+  top: design-token("space.1.5");
+  z-index: component-token("inputfield", "z-index");
+  min-width: design-token("space.3");
+  height: design-token("space.3");
+}
+
+.inputTextPlacementSm {
+  top: design-token("space.1");
+  min-width: design-token("space.2");
+  height: design-token("space.2");
+  font-size: design-token("font-size.75");
+}
+
+.inputTextPlacementLg {
+  top: design-token("space.2");
+}
+
+.textStart {
+  left: design-token("space.1.5");
+  color: design-token("color.neutral.600");
+}
+
+.textEnd {
+  right: design-token("space.1.5");
+  color: design-token("color.neutral.400");
```

Comment:
we should put the `font-size` and `color` on the `<Text>` instead the `InputText` component:

i.e.:
```
<Text variant={size === "sm" ? "body2" : "body1"} color={alignment === "start" ? "neutral.600" : "neutral.400">
```